types/index.d.ts

```diff
@@ -56,64 +58,64 @@ declare namespace graphqlHTTP {
     /**
      * A value to pass as the context to the graphql() function.
      */
-    context?: unknown;
+    context?: Maybe<unknown>;
 
     /**
      * An object to pass as the rootValue to the graphql() function.
      */
-    rootValue?: unknown;
+    rootValue?: Maybe<unknown>;
 
     /**
      * A boolean to configure whether the output should be pretty-printed.
      */
-    pretty?: boolean | null;
+    pretty?: Maybe<boolean>;
```

Comment:
One thing to note: after TS conversion we need to get rid of `Maybe` types, AFAIK they are not part of idiomatic TS and at that point would be just a Flow legacy.